Alignment/OfflineValidation/test/testPVValidation_Relvals_MC.ini

```diff
@@ -0,0 +1,35 @@
+###########################
+# test command:
+# python submitAllJobs.py -j TEST -D /RelValNuGun_UP15/CMSSW_8_0_0_pre2-PU25ns_76X_mcRun2_asymptotic_v12-v1/GEN-SIM-RECO -i testPVValidation_RelVals_MC.ini 
+
+[Job]
+jobname: test
+isda: True
+ismc: True
+maxevents: 10000
+
+[Conditions]
+gt: auto:run2_mc
+allFromGT: True
+applyextracond: True
+alignmentdb: frontier://FrontierProd/CMS_CONDITIONS 
+alignmenttag: TrackerAlignment_2015StartupPessimisticScenario_mc
+apedb: frontier://FrontierProd/CMS_CONDITIONS
+apetag: TrackerAlignmentExtendedErrors_2015StartupPessimisticScenario_mc
+applybows: True
+bowdb: frontier://FrontierProd/CMS_CONDITIONS
+bowtag: TrackerSurfaceDeformations_2011Realistic_v2_mc
+
+[Type]
+vertextype: offlinePrimaryVertices
+tracktype: generalTracks
+
+[Selection]
+applyruncontrol: True
+ptcut: 3.
+runboundary: None
+lumilist: None
+
+[ExtraConditions]    
+#TrackerAlignmentRcd: sqlite_file:/afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN2/HIP/aspiezia/CMSSW_7_4_0_pre6/src/Alignment/HIPAlignmentAlgorithm/CRUZET_ALL/alignments.db,Alignments
+#TrackerAlignmentErrorRcd: sqlite_file:/afs/cern.ch/user/m/musich/public/moduleDepenentAPE/TrackerAlignmentErr_2009_v2_express_SAFEAPEinPixel.db,TrackerAlignmentErr_2009_v2_express_SAFEAPEinPixel
```

Comment:
@mmusich do you maybe have a condition that works in general for MC to replace these comments?